guide/integration/index.md

```diff
@@ -49,15 +49,7 @@ https://fishjam.io/api/v1/connect/<YOUR_APP_UUID>/room-manager?roomName=foo&user
 Room Manager doesn't implement any participant authentication. Anyone
 using the same room name and user name will receive **the same Participant Token!**
 
-For production, make sure to **set up your own backend** using our Server SDKs and authenticate the client on your own!
-
-:::
-
-### How do I integrate my backend then?
-
-:::warning[LINKS UNDER CONSTRUCTION]
-
-Follow the links to see how to setup our SDK in **Python** or **NodeJS**.
+For production, make sure to [**set up your own backend**](../server.mdx) using our Server SDKs and authenticate the client on your own!
```

Comment:
I'm not sure if we really need to make it bolded/with `!`
```suggestion
For production environment, make sure to [set up your own backend](../server.mdx) using our Server SDKs and authenticate the client on your own.
```

Reply:
Sure, we can make it less aggressive, since it's already all red